pkg/trace/README.md

```diff
@@ -0,0 +1,22 @@
+# Trace
+
+This package have filter for generate traceID if no traceID in request header.
```

Comment:
"This package contains filter for generating `traceID` if there is no `traceID` in request header."

Reply:
Updated